test/image.test.js

```diff
@@ -35,6 +36,36 @@ describe('Image', function () {
     assert.strictEqual(320, img.height);
   });
 
+  it('test Image#onload multiple times', function() {
+    var img = new Image
+      , n = 0;
```

Comment:
Can you call this variable `onloadCalled` as in the other tests? This applies to all your uses of `n`.